front/src/applications/stdcmV2/components/StdcmOperationalPoint.tsx

```diff
@@ -148,13 +150,15 @@ const StdcmOperationalPoint = ({ updatePoint, point, disabled }: StdcmOperationa
           disabled={disabled}
         />
       </div>
-      <div className="suggestions stdcm-v2-ch-selector w-100 py-2 col-3">
-        <SelectSNCF
+      <div className="suggestions stdcm-v2-ch-selector w-100 pl-1 pb-2 col-3">
+        <Select
           label={t('trainPath.ch')}
           id="ch"
```

Comment:
As the component is used many times
```suggestion
          id={`${point?.id}-ch`}
```

Same for `StdcmSuggestions` at line 143